packages/components/src/modal/index.tsx

```diff
@@ -198,7 +198,13 @@ function UnforwardedModal(
 								) }
 							</div>
 						) }
-						{ children }
+						<div
```

Comment:
Hmm, let's not modify the Modal component. The problem only exists in the keyboard shortcuts modal, these changes are not needed globally, right?